frontend-project/src/pages/login/OAuthCallbackView.tsx

```diff
@@ -0,0 +1,35 @@
+import { Space, Spin } from 'antd';
+import React, { useEffect } from 'react';
+import { formatMessage } from 'umi-plugin-react/locale';
+import { useAuth } from '../../hooks/useAuth';
+import { localeKeys } from '../../locales/pl-PL';
+import { UsersService } from '../../services/users';
+import styles from './sign-in.less';
+
+export default function OAuthCallbackView() {
+  const { login } = localeKeys;
+  const auth = useAuth();
+  useEffect(() => {
+    const initialQueryParametrs = new URLSearchParams(window.location.search);
+    const paramsArray: [string, string][] = [];
+    initialQueryParametrs.forEach((value, key) => paramsArray.push([key, value]));
```

Comment:
Wydaje mi się, że nie potrzebna ta tablica pośrednia, a można po prostu stworzyć słownik i w nim to przypisać...Najprościej `Object.fromEntries(new URLSearchParams(window.location.search))`.